File: Sqrt.java
import java.util.Scanner;
public class Sqrt{
	public static void main(String[]args){
		Scanner sobjr=new Scanner(System.in);
		System.out.println("Enter base:");
		double dbase=sobjr.nextDouble();
		System.out.println("Square root of "+dbase+" is "+Math.sqrt(dbase));
}
}

File: sqrt.c
//programm to calculate square root of given number
#include<stdio.h>
#include<stdlib.h>
#include<math.h>
main()
{
	//sqrt -----double   sqrt(double)
	double dbase;
	system("cls");
	printf("Enter base:");
	scanf("%lf",&dbase);
	printf("square root of %.2lf is %.2lf\n",dbase,sqrt(dbase));
	system("\npause");
}
